resources/core/charts/cluster-users/values.yaml

```diff
@@ -21,6 +21,7 @@ clusterRoles:
       - "servicecatalog.k8s.io"
       - "settings.k8s.io"
       - "metrics.k8s.io"
+      - "coordination.k8s.io "
```

Comment:
Why this is added?

Reply:
This is a "by-the-way" fix since k8s 1.16 a new group is required to perform `kubectl describe node`